package.json

```diff
@@ -16,19 +16,66 @@
     "url": "https://github.com/Microsoft/vscode-python/issues"
   },
   "engines": {
-    "vscode": "^1.46.0"
+    "vscode": "^1.48.0"
   },
   "categories": [
     "Other"
```

Comment:
We should add some categories here. Seems like most of the existing Jupyter ones fit.